### PR TITLE
Cater for potential preference for runit

### LINKED ISSUE #70: Support runit in non-production environments
Developers using Docker for development/testing/staging environment (not for production) would benefit from using the same scripts for all environments. In Docker containers, alternative init systems are usually preferred to systemd - runit is one of those e.g. see https://github.com/phusion/baseimage-docker .

### PR DESCRIPTION
**Pull request by lucafavatella**
Closes #70.

Opened issue while having the PR already ready... because when opening the PR I read the [guidelines](https://github.com/basho-labs/ansible-riak/blob/6c9ede59d22a0ed96d5fdcb475449532bc44434a/CONTRIBUTING.md#how-to-contribute-to-the-ansible-riak-role).

### CODE REVIEW

**Comment by lucafavatella** [comment]
The Travis CI job failed but I am not sure why as its log cannot be fetched. It may be related to recent AWS S3 issues.

**Comment by christophermancini** [comment]
@lucafavatella I restarted the test run.

**Comment by lucafavatella** [comment]
@christophermancini In build 101: "There was an error while trying to fetch the log."

**Comment by christophermancini** [comment]
@lucafavatella Yea, I will attempt to run your changes locally to see if the test playbook works.

File: files/etc_service_riak_run [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+exec /sbin/setuser riak "$(ls -d /usr/lib/riak/erts*)/bin/run_erl" "/tmp/riak" \
```
Comment (lucafavatella) [inline files/etc_service_riak_run:2]: `/sbin/setuser` is not a valid command.

**Comment by lucafavatella** [comment]
This PR as it is cannot be merged because of usage of custom command `/sbin/setuser`. See discussion in #70 for next steps.

**Pull request closed without merging**